samples/client/petstore/python-experimental/test/__init__.py

```diff
@@ -0,0 +1,81 @@
+# coding: utf-8
```

Comment:
Please remove this file from python-experimental because we are not yet using it

Reply:
Can you resolve this conversation?